src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.exceptions.InvalidFruitException;
+import core.basesyntax.service.exceptions.InvalidQuantityException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class Storage {
+    private static final Map<Fruit, Integer> fruits = new HashMap<>();
+
+    public static Integer getQuantity(Fruit fruit) {
+        if (fruits.get(fruit) != null) {
+            return fruits.get(fruit);
+        }
+        throw new InvalidQuantityException("Not found quantity value for fruit: " + fruit);
+    }
+
+    public static void put(Fruit fruit, Integer quantity) {

```

Comment:
The method 'put' is a static method. This implies that it can be accessed without creating an instance of the class. This can lead to problems with object-oriented design, consider removing the static modifier.